include/deal.II/base/vectorization.h

```diff
@@ -3941,6 +3941,167 @@ operator<<(std::ostream &out, const VectorizedArray<Number, width> &p)
   return out;
 }
 
+
+/**
+ * @name Ternary operations on VectorizedArray
+ */
+//@{
+
+/**
+ * TODO
+ */
+enum SIMDComparison : int
```

Comment:
I have created an enum with more legible names for the most useful variants.
I have chosen to only use `ordered` `non-signaling` variants here.

Reply:
Want to make it an `enum class` instead?

Reply:
I have changed this to an `enum class`.